src/js/visual/image.js

```diff
@@ -269,6 +269,7 @@ ripe.Image.prototype._registerHandlers = function() {
 ripe.Image.prototype._unregisterHandlers = function() {
     if (this.loadListener) this.element.removeEventListener("load", this.loadListener);
     if (this.errorListener) this.element.removeEventListener("error", this.errorListener);
+    if (this._loadedCallback) this._loadedCallback({ canceled: true });

```

Comment:
@gcandal This is the fix, the remaining changes were Pretty's fault

Reply:
@NFSS10 please revert them. I don't know what's going on, but clearly there shouldn't be a space between `function` and `()`.